pandas/tools/pivot.py

```diff
@@ -170,6 +170,9 @@ def pivot_table(data, values=None, index=None, columns=None, aggfunc='mean',
     if len(index) == 0 and len(columns) > 0:
         table = table.T
 
+    if isinstance(table, Series):
```

Comment:
Your solution works for the specific scenario in #4386, but a `Series`is still returned regardless when [pivoting with only the columns argument](https://github.com/pandas-dev/pandas/blob/master/pandas/tools/tests/test_pivot.py#L351) or [pivoting with no values or cols passed](https://github.com/pandas-dev/pandas/blob/master/pandas/tools/tests/test_pivot.py#L486) due to the `groupby`  and `unstack` operations respectively. In these cases, I think `to_frame()` might be necessary if `pivot_table` should always return a `DataFrame`.

Or should I just fix the specific scenario in #4386 and change the docs to mention that `pivot_table` can return a `Series`?